libs/generic-view/feature/src/lib/setup-component.tsx

```diff
@@ -79,17 +79,15 @@ export const setupComponent = <P extends object>(
       }) as EntityData
     })
 
-    const editableProps = {
-      ...dataProps,
-      config: {
-        ...config,
-      },
-      data: undefined as unknown,
-    }
+    const editableProps = JSON.parse(
+      JSON.stringify({
```

Comment:
Hey! I noticed you're using `JSON.stringify/parse` here — are you aiming for a deep copy? If so, maybe `_.cloneDeep()` would be clearer and more predictable in terms of intent. What do you think?

Reply:
Good point. Changed in [34c1131](https://github.com/mudita/mudita-center/pull/2076/commits/34c113113a5c96cbee59a520d9a1f43401af6e57)